
2022-06-27 13:52:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS PARTITA (NASCOSTO BOOLEAN, STANZACORRENTE INT, VIVO BOOLEAN,OGGETTOINVETARIO1 VARCHAR(20,[*] OGGETTOINVETARIO2 VARCHAR(20, OGGETTOINVETARIO3 VARCHAR(20, OGGETTOINVETARIO4 VARCHAR(20,)OGGETTOINVETARIO5 VARCHAR(20), OGGETTOINVETARIO6 VARCHAR(20), STANZAVISITATA1 BOOLEAN, STANZAVISITATA2 BOOLEAN), STANZAVISITATA3 BOOLEAN,STANZAVISITATA4 BOOLEAN, STANZAVISITATA5 BOOLEAN, STANZAVISITATA6 BOOLEAN, STANZAVISITATA7 BOOLEAN, STANZAVISITATA8 BOOLEAN, STANZAVISITATA9 BOOLEAN,STANZAVISITATA10 BOOLEAN, STANZAVISITATA11 BOOLEAN, STANZAVISITATA12 BOOLEAN)"; expected ")"; SQL statement:
CREATE TABLE IF NOT EXISTS partita (nascosto BOOLEAN, stanzaCorrente INT, Vivo BOOLEAN,OggettoInvetario1 VARCHAR(20, OggettoInvetario2 VARCHAR(20, OggettoInvetario3 VARCHAR(20, OggettoInvetario4 VARCHAR(20,)OggettoInvetario5 VARCHAR(20), OggettoInvetario6 VARCHAR(20), StanzaVisitata1 BOOLEAN, StanzaVisitata2 BOOLEAN), StanzaVisitata3 BOOLEAN,StanzaVisitata4 BOOLEAN, StanzaVisitata5 BOOLEAN, StanzaVisitata6 BOOLEAN, StanzaVisitata7 BOOLEAN, StanzaVisitata8 BOOLEAN, StanzaVisitata9 BOOLEAN,StanzaVisitata10 BOOLEAN, StanzaVisitata11 BOOLEAN, StanzaVisitata12 BOOLEAN) [42001-200]
2022-06-27 13:52:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS PARTITA (NASCOSTO BOOLEAN, STANZACORRENTE INT, VIVO BOOLEAN,OGGETTOINVETARIO1 VARCHAR(20,[*] OGGETTOINVETARIO2 VARCHAR(20, OGGETTOINVETARIO3 VARCHAR(20, OGGETTOINVETARIO4 VARCHAR(20,)OGGETTOINVETARIO5 VARCHAR(20), OGGETTOINVETARIO6 VARCHAR(20), STANZAVISITATA1 BOOLEAN, STANZAVISITATA2 BOOLEAN), STANZAVISITATA3 BOOLEAN,STANZAVISITATA4 BOOLEAN, STANZAVISITATA5 BOOLEAN, STANZAVISITATA6 BOOLEAN, STANZAVISITATA7 BOOLEAN, STANZAVISITATA8 BOOLEAN, STANZAVISITATA9 BOOLEAN,STANZAVISITATA10 BOOLEAN, STANZAVISITATA11 BOOLEAN, STANZAVISITATA12 BOOLEAN)"; expected ")"; SQL statement:
CREATE TABLE IF NOT EXISTS partita (nascosto BOOLEAN, stanzaCorrente INT, Vivo BOOLEAN,OggettoInvetario1 VARCHAR(20, OggettoInvetario2 VARCHAR(20, OggettoInvetario3 VARCHAR(20, OggettoInvetario4 VARCHAR(20,)OggettoInvetario5 VARCHAR(20), OggettoInvetario6 VARCHAR(20), StanzaVisitata1 BOOLEAN, StanzaVisitata2 BOOLEAN), StanzaVisitata3 BOOLEAN,StanzaVisitata4 BOOLEAN, StanzaVisitata5 BOOLEAN, StanzaVisitata6 BOOLEAN, StanzaVisitata7 BOOLEAN, StanzaVisitata8 BOOLEAN, StanzaVisitata9 BOOLEAN,StanzaVisitata10 BOOLEAN, StanzaVisitata11 BOOLEAN, StanzaVisitata12 BOOLEAN) [42001-200]
2022-06-27 13:53:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS PARTITA (NASCOSTO BOOLEAN, STANZACORRENTE INT, VIVO BOOLEAN,OGGETTOINVETARIO1 VARCHAR(20,[*] OGGETTOINVETARIO2 VARCHAR(20, OGGETTOINVETARIO3 VARCHAR(20, OGGETTOINVETARIO4 VARCHAR(20,)OGGETTOINVETARIO5 VARCHAR(20), OGGETTOINVETARIO6 VARCHAR(20), STANZAVISITATA1 BOOLEAN, STANZAVISITATA2 BOOLEAN), STANZAVISITATA3 BOOLEAN,STANZAVISITATA4 BOOLEAN, STANZAVISITATA5 BOOLEAN, STANZAVISITATA6 BOOLEAN, STANZAVISITATA7 BOOLEAN, STANZAVISITATA8 BOOLEAN, STANZAVISITATA9 BOOLEAN,STANZAVISITATA10 BOOLEAN, STANZAVISITATA11 BOOLEAN, STANZAVISITATA12 BOOLEAN)"; expected ")"; SQL statement:
CREATE TABLE IF NOT EXISTS partita (nascosto BOOLEAN, stanzaCorrente INT, Vivo BOOLEAN,OggettoInvetario1 VARCHAR(20, OggettoInvetario2 VARCHAR(20, OggettoInvetario3 VARCHAR(20, OggettoInvetario4 VARCHAR(20,)OggettoInvetario5 VARCHAR(20), OggettoInvetario6 VARCHAR(20), StanzaVisitata1 BOOLEAN, StanzaVisitata2 BOOLEAN), StanzaVisitata3 BOOLEAN,StanzaVisitata4 BOOLEAN, StanzaVisitata5 BOOLEAN, StanzaVisitata6 BOOLEAN, StanzaVisitata7 BOOLEAN, StanzaVisitata8 BOOLEAN, StanzaVisitata9 BOOLEAN,StanzaVisitata10 BOOLEAN, StanzaVisitata11 BOOLEAN, StanzaVisitata12 BOOLEAN) [42001-200]
2022-06-27 13:54:31 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 16 more
2022-06-27 13:56:50 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.MVStore.getUnsupportedWriteFormatException(MVStore.java:1066)
	at org.h2.mvstore.MVStore.readStoreHeader(MVStore.java:886)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:463)
	... 16 more
2022-06-27 13:56:50 database: opening /home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale
org.h2.message.DbException: Unsupported database file version or invalid file header in file "/home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db" [90048-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:161)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Unsupported database file version or invalid file header in file "/home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db" [90048-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 17 more
Caused by: org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.MVStore.getUnsupportedWriteFormatException(MVStore.java:1066)
	at org.h2.mvstore.MVStore.readStoreHeader(MVStore.java:886)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:463)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 14 more
2022-06-29 20:30:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [21002-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.dml.Insert.prepare(Insert.java:319)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:122)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:33:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "22" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setBoolean(JdbcPreparedStatement.java:585)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:151)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:33:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#4" is not set; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:168)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:35:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "22" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setBoolean(JdbcPreparedStatement.java:585)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:152)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:35:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#4" is not set; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:169)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:36:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "22" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setBoolean(JdbcPreparedStatement.java:585)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:151)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:36:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#4" is not set; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:168)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:36:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "22" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setBoolean(JdbcPreparedStatement.java:585)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:151)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:36:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#4" is not set; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:168)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:37:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "22" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setBoolean(JdbcPreparedStatement.java:585)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:151)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:37:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#4" is not set; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:168)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:38:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "22" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setBoolean(JdbcPreparedStatement.java:585)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:151)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-29 20:42:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Parameter "#4" is not set; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [90012-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.Parameter.checkSet(Parameter.java:83)
	at org.h2.command.Prepared.checkParameters(Prepared.java:182)
	at org.h2.command.CommandContainer.update(CommandContainer.java:187)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at GestioneSalvataggio.inserimentoSalvataggioInTabella(GestioneSalvataggio.java:168)
	at GameManager.salvaPartita(GameManager.java:1129)
	at Parser.parser(Parser.java:69)
	at Test.main(Test.java:30)
2022-06-30 11:24:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 11:25:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 11:28:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 11:28:28 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.MVStore.getUnsupportedWriteFormatException(MVStore.java:1066)
	at org.h2.mvstore.MVStore.readStoreHeader(MVStore.java:886)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:463)
	... 16 more
2022-06-30 11:28:28 database: opening /home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale
org.h2.message.DbException: Unsupported database file version or invalid file header in file "/home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db" [90048-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:161)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Unsupported database file version or invalid file header in file "/home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db" [90048-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 17 more
Caused by: org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.MVStore.getUnsupportedWriteFormatException(MVStore.java:1066)
	at org.h2.mvstore.MVStore.readStoreHeader(MVStore.java:886)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:463)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 14 more
2022-06-30 11:28:29 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.MVStore.getUnsupportedWriteFormatException(MVStore.java:1066)
	at org.h2.mvstore.MVStore.readStoreHeader(MVStore.java:886)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:463)
	... 16 more
2022-06-30 11:28:29 database: opening /home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale
org.h2.message.DbException: Unsupported database file version or invalid file header in file "/home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db" [90048-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:161)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Unsupported database file version or invalid file header in file "/home/vito/Scrivania/Esame-MAP/Gioco_MAP/src/main/resources/dataBaseLocale.mv.db" [90048-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 17 more
Caused by: org.h2.mvstore.MVStoreException: The write format 1 is smaller than the supported format 2 [2.1.214/5]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.MVStore.getUnsupportedWriteFormatException(MVStore.java:1066)
	at org.h2.mvstore.MVStore.readStoreHeader(MVStore.java:886)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:463)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 14 more
2022-06-30 11:30:15 jdbc[5]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 11:35:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 11:39:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PARITA" not found; SQL statement:
SELECT
NASCOSTO,
STANZACORRENTE,
VIVO,
OGGETTOINVETARIO1,  
OGGETTOINVETARIO2,
OGGETTOINVETARIO3, 
OGGETTOINVETARIO4, 
OGGETTOINVETARIO5, 
OGGETTOINVETARIO6,
STANZAVISITATA1,  
STANZAVISITATA2,  
STANZAVISITATA3 ,
STANZAVISITATA4, 
STANZAVISITATA5,  
STANZAVISITATA6,  
STANZAVISITATA7,  
STANZAVISITATA8,  
STANZAVISITATA9 ,
STANZAVISITATA10 , 
STANZAVISITATA11 , 
STANZAVISITATA12   FROM parita [42102-200]
2022-06-30 11:40:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PARITA" not found; SQL statement:
SELECT
NASCOSTO,
STANZACORRENTE,
VIVO,
OGGETTOINVETARIO1,  
OGGETTOINVETARIO2,
OGGETTOINVETARIO3, 
OGGETTOINVETARIO4, 
OGGETTOINVETARIO5, 
OGGETTOINVETARIO6,
STANZAVISITATA1,  
STANZAVISITATA2,  
STANZAVISITATA3 ,
STANZAVISITATA4, 
STANZAVISITATA5,  
STANZAVISITATA6,  
STANZAVISITATA7,  
STANZAVISITATA8,  
STANZAVISITATA9 ,
STANZAVISITATA10 , 
STANZAVISITATA11 , 
STANZAVISITATA12   FROM parita [42102-200]
2022-06-30 11:40:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PARITA" not found; SQL statement:
SELECT
NASCOSTO,
STANZACORRENTE,
VIVO,
OGGETTOINVETARIO1,  
OGGETTOINVETARIO2,
OGGETTOINVETARIO3, 
OGGETTOINVETARIO4, 
OGGETTOINVETARIO5, 
OGGETTOINVETARIO6,
STANZAVISITATA1,  
STANZAVISITATA2,  
STANZAVISITATA3 ,
STANZAVISITATA4, 
STANZAVISITATA5,  
STANZAVISITATA6,  
STANZAVISITATA7,  
STANZAVISITATA8,  
STANZAVISITATA9 ,
STANZAVISITATA10 , 
STANZAVISITATA11 , 
STANZAVISITATA12   FROM parita [42102-200]
2022-06-30 11:40:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 11:41:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 11:57:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:06:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:27:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:33:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:38:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:43:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:43:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:48:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: The database is open in exclusive mode; can not open additional connections [90135-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Session.checkSuspended(Session.java:1369)
	at org.h2.engine.Session.transitionToState(Session.java:1361)
	at org.h2.engine.Session.waitIfExclusiveModeEnabled(Session.java:1633)
	at org.h2.command.Command.executeQuery(Command.java:186)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:85)
	at GestioneSalvataggio.caricaSalvataggio(GestioneSalvataggio.java:215)
	at GameManager.caricaSalvataggio(GameManager.java:1105)
	at Parser.parser(Parser.java:64)
	at Test.main(Test.java:30)
2022-06-30 12:48:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 12:56:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 13:01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 13:08:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 13:12:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 13:16:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 13:17:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 13:18:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-200]
2022-06-30 14:40:14 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "IF[*] EXISTS DROP TABLE PARTITA"; SQL statement:
IF EXISTS DROP TABLE partita [42000-200]
2022-06-30 14:40:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "IF[*] EXISTS DROP TABLE PARTITA"; SQL statement:
IF EXISTS DROP TABLE partita [42000-200]
2022-06-30 14:45:18 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PARTITA" not found; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [42102-200]
2022-06-30 19:09:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "EVENTO" not found; SQL statement:
SELECT
NASCOSTO,
STANZACORRENTE,
VIVO,
EVENTO, 
OGGETTOINVETARIO1,  
OGGETTOINVETARIO2,
OGGETTOINVETARIO3, 
OGGETTOINVETARIO4, 
OGGETTOINVETARIO5, 
OGGETTOINVETARIO6,
STANZAVISITATA1,  
STANZAVISITATA2,  
STANZAVISITATA3 ,
STANZAVISITATA4, 
STANZAVISITATA5,  
STANZAVISITATA6,  
STANZAVISITATA7,  
STANZAVISITATA8,  
STANZAVISITATA9 ,
STANZAVISITATA10 , 
STANZAVISITATA11 , 
STANZAVISITATA12   FROM partita [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at GestioneSalvataggio.caricaSalvataggio(GestioneSalvataggio.java:231)
	at GameManager.caricaSalvataggio(GameManager.java:1376)
	at Parser.parser(Parser.java:64)
	at Test.main(Test.java:30)
2022-06-30 19:11:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PARTITA" not found; SQL statement:
DROP TABLE partita [42102-200]
2022-06-30 19:11:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PARTITA" not found; SQL statement:
INSERT INTO partita VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [42102-200]
